apache/gravitino
# [4958] feat(iceberg): support event listener for Iceberg REST server

### What changes were proposed in this pull request?

1.   Integrate Event listener system to Iceberg REST server
2.  only dispatch create table event like `CreateIcebergTableEvent`  `CreateIcebergTablePreEvent`  `CreateIcebergTableFailureEvent` 


### Why are the changes needed?

Fix: #4958 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
1. add configuration to add `IcebergEventLogger`  in event listener
```
gravitino.eventListener.names = iceberg
gravitino.eventListener.iceberg.class = org.apache.gravitino.iceberg.extension.IcebergEventLogger
```
2. run query will see event logs.
